example/lib/src/platform_menu.dart

```diff
@@ -20,7 +20,7 @@ class _AppPlatformMenuState extends State<AppPlatformMenu> {
     }
 
     return PlatformMenuBar(
-      menus: <MenuItem>[
```

Comment:
actually, it's probably better to simply drop the type from here to keep it compatible with 3.3.x